src/ocean/util/container/more/Stack.d

```diff
@@ -133,7 +133,7 @@ struct Stack (V, int Size = 0)
                           if (depth < stack.length)
```

Comment:
These all `if (...)` can be converted to enforce for a nicer code.

Reply:
Full rewrite of module code style is my next step :)